### PR TITLE
21122 - Add css to disable to disable pointer events (highlight element on tabbing)

### PR DESCRIPTION
**Pull request by ketaki-deodhar**
*Issue #:* /bcgov/entity#21122

*Description of changes:*

added css to disable to disable pointer events (highlight element on tabbing)

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the business-filings-ui license (Apache 2.0).

### CODE REVIEW

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://business-filings-dev--pr-658-z9damay2.web.app

File: src/components/Dashboard/FilingHistoryList/FilingTemplate.vue [hunk 1]
```diff
@@ -1,6 +1,6 @@
 <template>
   <v-expansion-panel class="filing-template">
-    <v-expansion-panel-header>
+    <v-expansion-panel-header tabindex="-1">
```
Comment (ketaki-deodhar) [inline src/components/Dashboard/FilingHistoryList/FilingTemplate.vue:3]: adding this here disables highlighting the panel on tabbing

**Review by JazzarKarim — APPROVED** [review]
Nice!

File: src/components/Dashboard/FilingHistoryList/HeaderActions.vue [hunk 2]
```diff
@@ -7,6 +7,7 @@
       outlined
       color="primary"
       :ripple="false"
+      tabindex="-1"
```
Comment (severinbeauvais) [inline src/components/Dashboard/FilingHistoryList/HeaderActions.vue:10]: Why do we want to remove this from the tab order?

Comment (severinbeauvais) [reply src/components/Dashboard/FilingHistoryList/HeaderActions.vue:10]: Same question for the two other buttons in the other files.

Comment (ketaki-deodhar) [reply src/components/Dashboard/FilingHistoryList/HeaderActions.vue:10]: I removed the `v-expansion-panel` from tab order first but the buttons were still were in the tab order. See below

image


https://github.com/bcgov/business-filings-ui/assets/116035339/a872973b-7fef-41c0-8615-aae4cc90ff6f

**Review by ArwenQin — APPROVED** [review]
It works. LGTM

File: src/components/Dashboard/FilingHistoryList/HeaderActions.vue (hunk 2 above)
Comment (severinbeauvais) [reply src/components/Dashboard/FilingHistoryList/HeaderActions.vue:10]: This is an extension of "hide the collapse button for incomplete IAs/registrations/etc" change, right?

I think the view button(s) should still be visible but the hide buttons should not (nor be tabbable).

Comment (severinbeauvais) [reply src/components/Dashboard/FilingHistoryList/HeaderActions.vue:10]: Using the preview link above, I can still tab to the panel header and press Space or Enter to toggle it. That's what we want to get rid of.

tabbed to panel header
image

Comment (ketaki-deodhar) [reply src/components/Dashboard/FilingHistoryList/HeaderActions.vue:10]: I used the same link but I can't tab to the panel. This is weird!

https://github.com/bcgov/business-filings-ui/assets/116035339/765bb436-1f01-48a2-b15c-65326a413356

Comment (severinbeauvais) [reply src/components/Dashboard/FilingHistoryList/HeaderActions.vue:10]: Click the View button to expand the panel, then try tabbing (or shift-tabbing).

File: src/components/Dashboard/FilingHistoryList/FilingTemplate.vue [hunk 3]
```diff
@@ -1,5 +1,8 @@
 <template>
-  <v-expansion-panel class="filing-template">
+  <v-expansion-panel
+    class="filing-template py-5 px-6"
+    :disabled="true"
```
Comment (ketaki-deodhar) [inline src/components/Dashboard/FilingHistoryList/FilingTemplate.vue:4]: disabling the `v-expansion-panel` resolves the issue of tabbing.

File: src/components/Dashboard/FilingHistoryList/FilingTemplate.vue [hunk 4]
```diff
@@ -180,11 +183,6 @@ export default class FilingTemplate extends Vue {
 <style lang="scss" scoped>
 @import "@/assets/styles/theme.scss";
 
-.v-expansion-panel {
-  padding: 1.25rem 1.5rem;
-  pointer-events: none; // disable expansion generally
-}
-
```
Comment (ketaki-deodhar) [inline src/components/Dashboard/FilingHistoryList/FilingTemplate.vue:187]: removed this from css and added the class for padding in the template (line 3)

File: src/components/Dashboard/FilingHistoryList/FilingTemplate.vue (hunk 3 above)
Comment (severinbeauvais) [reply src/components/Dashboard/FilingHistoryList/FilingTemplate.vue:4]: @kialj876 @cameron-eyds @seeker25 Here is something you might use. By disabling the panel here, you prevent the user from tabbing to the panel header (which we don't want) and encourages them to tab/click to the action buttons (which we do want).

File: src/components/Dashboard/FilingHistoryList/HeaderActions.vue (hunk 2 above)
Comment (severinbeauvais) [reply src/components/Dashboard/FilingHistoryList/HeaderActions.vue]: This code change was reverted so this comment is obsolete. I'm resolving it.

**Pull request merged**